docs/pipeline.rst

```diff
@@ -66,6 +66,19 @@ pushes, this is all handled by the pipeline, and is subject to change according
     #. If any staging tests fail above, check retries, e.g. `bedrock_test_stage_eu_west`_
 #. Tag and push the deployment by running ``bin/tag-release.sh --push``
 
+.. note::
+
+    By default the ``tag-release.sh`` script will push to the ``origin`` git remote. If you'd
+    like for it to push to a different remote name you can either pass in a ``-r`` or
+    ``--remote`` argument, or set the ``MOZ_GIT_REMOTE`` environment variable. So the following
```

Comment:
hmm... possible, but probably overkill for this. Easiest thing to do is to set an alias for your command in your `.bashrc` file:

```bash
alias bedrock-push='bin/tag-release.sh --push -r upstream`
```

Then you just have to type `bedrock-push` whenever you want to do a push. Alternately, just set the env var in your `.bashrc` (`export MOZ_GIT_REMOTE=upstream`).